packages/model-viewer/src/three-components/ARRenderer.ts

```diff
@@ -108,10 +176,8 @@ export class ARRenderer extends EventDispatcher {
     // TODO: this method should be added to three.js's exported interface.
     (this.threeRenderer as any)
         .setFramebuffer(session.renderState.baseLayer!.framebuffer);
-    this.threeRenderer.setSize(
-        session.renderState.baseLayer!.framebufferWidth,
-        session.renderState.baseLayer!.framebufferHeight,
-        false);
+    (this[$presentedScene]!.element as
+     ModelViewerElement)[$onResize](window.screen);
```

Comment:
This cast shouldn't be ncessary, `$onResize` is available on `ModelViewerElementBase`: 

https://github.com/google/model-viewer/blob/239323cac1b8ff000d6c201e980022def3aef98e/packages/model-viewer/src/model-viewer-base.ts#L523-L526